ui-tests/pom.xml

```diff
@@ -16,7 +16,7 @@
   <name>UI Tests of Warnings Plugin</name>
 
   <properties>
-    <jenkins.version>2.361</jenkins.version>
+    <jenkins.version>2.362</jenkins.version>
```

Comment:
Sorry but we would like this for 2.361 :smile: We want to do PCT testing of the next LTS release.

Reply:
So you need a single release with this version? My UI tests are typically running vs the latest Jenkins baseline.

Reply:
Not sure this matters—I think this module would be ignored by PCT anyway?

Reply:
Thank you, I'm not sure it matters either but a single release on 2.361 would be the safest option. I'll test this out in https://github.com/jenkinsci/bom/pull/1384 once it's released (can't test before that due to lack of incremental).

Reply:
I think incrementals does not work with plugins that have UI tests. I also would assume that it is sufficient to use the `plugin` submodule for the tests in PCT. For all my maven commands I am using `cd plugin` before I invoke them.